cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -195,6 +195,7 @@ func getAPIServerCommand(cfg *kubeadmapi.MasterConfiguration) []string {
 		defaultArguments["endpoint-reconciler-type"] = kubeadmconstants.LeaseEndpointReconcilerType
 	}
 
+	// This feature is enabled by default in kubeadm, you can disabled it manually.
```

Comment:
Being dynamic kubelet GA and enabled by default, could you kindly confirm that it is still the case we should set `DynamicKubeletConfig=true`. I was expecting the logic here to be changed (set false when we don't want dynamic kubelet)....